### PR TITLE
i18n updates

### PR DESCRIPTION
**Pull request by razbakov**
- Feed
- Calendar
- form fields refactoring
- style of i18n files

### CODE REVIEW

File: components/MainNavigation.vue [hunk 1]
```diff
@@ -31,15 +31,20 @@
         <span>{{ $t('myprofile.title') }}</span>
       </TButton>
       <TButton to="/settings" type="nav" :label="$t('settings.title')" />
-      <TButton to="/signout" type="nav" :label="$t('signout')" />
+      <TButton to="/signout" type="nav" :label="$t('auth.signout')" />
     </template>
     <template v-else>
-      <TButton allow-guests to="/signin" type="nav" :label="$t('signin')" />
```
Comment (best-lucky1030) [inline components/MainNavigation.vue:37]: got it why you mentioned about `Auth`...

File: pages/feed.vue [hunk 2]
```diff
@@ -36,27 +38,16 @@ import { useCities } from '~/use/cities'
...
-    const filterTypeList = [
-      { label: 'Newest', value: 'Newest' },
-      { label: 'Hot', value: 'Hot' },
-      { label: 'Popular', value: 'Popular' },
-      { label: 'Watching', value: 'Watching' },
-      { label: 'Starred', value: 'Starred' },
-      { label: 'Archived', value: 'Archived' },
-      { label: 'Yours', value: 'Authored' },
-    ]
+    const { postTypeList, filterTypeList } = usePosts()
```
Comment (best-lucky1030) [inline pages/feed.vue:50]: where `filterTypeList` is from?

Comment (razbakov) [reply pages/feed.vue:50]: `usePosts()` returns object. This way you can access its properties. Read more about Object destructuring in javascript.

Comment (razbakov) [reply pages/feed.vue:50]: Comments are not for discussions. They are to request changes. Every comment considered to be an issue and requires revision. Please use Slack for discussions.

**Pull request merged**